locust/web.py

```diff
@@ -332,7 +332,17 @@ def failures_stats_csv() -> Response:
         def request_stats() -> Response:
             stats: List[Dict[str, Any]] = []
             errors: List[StatsErrorDict] = []
-
+            options = self.environment.parsed_options
```

Comment:
Doesn't `environment` work? (instead of `self.environment`). But anyways, like I mentioned in my PR comment, I think this should be a "hidden" parameter, like with the other stats.py settings (like this one https://github.com/locustio/locust/blob/5cf5da22b6774a537e14b08ed68acb9dfc4a8e98/locust/stats.py#L134). Would that be ok for you?

Reply:
Does "hidden" parameter mean, should the parameter be outside of WebUI class?  or is it ok as instance variable under def __init__

like 
![image](https://user-images.githubusercontent.com/79035645/222918378-77a6d9e2-2ddc-407d-85e9-675a0eb269e1.png)
web.py

Reply:
Hidden in the sense that it is just a variable on the stats module, set from inside the locustfile, not an argument. As i linked here: https://docs.locust.io/en/stable/configuration.html#customization-of-statistics-settings

Reply:
Let me clarify, 

stats.py has default value like this,

![image](https://user-images.githubusercontent.com/79035645/223045389-a29aab5c-3fb5-4914-b55f-7bf9b9be3669.png)

then if command line argument percentiles have value, the parameter is overwritten like this when it is used.
![image](https://user-images.githubusercontent.com/79035645/223045780-8da0bc31-d40f-40b2-992b-2a8e90fa1b8e.png)

Is it what you expect?

Reply:
Almost :) what I mean is dont have a command line parameter at all, and only expose it on stats.py. There are too many command line options already, and there needs to be some effort in cleaning them up before adding new ones (maybe grouping all stats-related stuff in an ”advanced” block somehow? The --help text is already ridiculously long, and I dont want to make it longer.

Reply:
Thank you for the explanation, I got what you intend. I thought command line argument is easier to change config test by test. But anyway, I have removed command line arguments of percentiles and percentiles values for response time chart is from stats.PERCENTILES_TO_CHART param. I have pushed the code, please review it.